owasp-top10-2021-apps/a9/games-irados/app/routes.py

```diff
@@ -78,8 +80,10 @@ def login():
         user_password, success = database.get_user_password(username)
         if not success or user_password == None or not psw.validate_password(str(user_password[0])):
             flash("Usuario ou senha incorretos", "danger")
+            app.logger.info("Unsuccessful login attempt. Username: {}; Database success: {};".format(username, success))
```

Comment:
Here, `success` will be always set to `False`, right? It would be also be nice to have the user IP address (we could monitor for credential spraying attacks, for example)

Reply:
fixed